exercises/chapter4/test/Examples.purs

```diff
@@ -7,14 +7,14 @@ import Data.Foldable (product)
 import Data.Maybe (fromMaybe)
 import Data.Path (Path, ls)
 
--- ANCHOR: fact
```

Comment:
Don't want this to be confused with the other **fact**oring functions in this chapter